### PR TITLE
ENG-3803 Fixed base with updated node.

### PR DESCRIPTION
**Pull request by jgrim**
https://jira.thegrommet.com/browse/ENG-3803

### CODE REVIEW

File: php-base/Dockerfile [hunk 1]
```diff
@@ -2,14 +2,16 @@ FROM php:7.3-fpm
 LABEL description="The Grommet PHP FPM Base"
 LABEL maintainer="todd@thegrommet.com"
 
-RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
```
Comment (aredridel) [inline php-base/Dockerfile:5]: Do we want to move to 14? It's not LTS status yet.

Comment (aredridel) [reply php-base/Dockerfile:5]: (close though from what I hear)

Comment (jgrim) [reply php-base/Dockerfile:5]: You're right. It apparently broke some of the automated tests for Lucian. I'm rolling it back to 12 now.

Comment (aredridel) [reply php-base/Dockerfile]: Yeah. We're still relying on a couple modules that have more strict and LTS-only dependencies and that pins us until the ecosystem updates. It's unfortunate but improving over time.

**Pull request merged**